src/JWT/JwtValidator.cs

```diff
@@ -3,9 +3,15 @@
 using System.Linq;
 using JWT.Algorithms;
 using JWT.Exceptions;
-
+using JWT.Internal;
 using static JWT.Internal.EncodingHelper;
 
+#if NET35
+using IPayloadData = System.Collections.Generic.IDictionary<string, object>;

```

Comment:
Could it have a little more nice name? Maybe just `IDictionary` or `StringObjectDictionary`?

Reply:
`IReadOnlyPayloadDictionary`?